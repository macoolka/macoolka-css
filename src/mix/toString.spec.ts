import {parse}from './index';
describe('parse module', () => {
     it('toString',()=>{
        expect(parse({
            mkHeight:'full',
            mkFlexAlign: 'center',
            mkMedia:[{mkColor:'surface',width:200},{mkColor:'body',width:300},{mkColor:'body',width:500},{fontSize:'40px',width:700}],
        })).toMatchSnapshot();
    })
     it('parse media', () => {
        expect(parse({})).toEqual('\n\n')
        expect(parse({ mkMedia: [{ width: '10px' }, { width: '100%' }, { width: '300px' }] })).
            toEqual(
`
@media screen and (max-width: 93em) {
  width: 300px;
}
@media screen and (max-width: 80em) {
  width: 100%;
}
@media screen and (max-width: 50em) {
  width: 10px;
}
`                )
    }) 
});


