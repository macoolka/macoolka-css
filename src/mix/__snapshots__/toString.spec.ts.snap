// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse module toString 1`] = `
"height: 100%;
justify-content: center;
@media screen and (max-width: 120em) {
  font-size: 40px;
  width: 700px;
}
@media screen and (max-width: 93em) {
  width: 500px;
  background-color: #fafafa;
  color: rgba(0, 0, 0, 0.87);
}
@media screen and (max-width: 80em) {
  width: 300px;
  background-color: #fafafa;
  color: rgba(0, 0, 0, 0.87);
}
@media screen and (max-width: 50em) {
  width: 200px;
  background-color: #fff;
  color: rgba(0, 0, 0, 0.87);
}
"
`;
